Indico/Mutation/ModelGroupLoad.cs

```diff
@@ -31,8 +47,8 @@ public ModelGroupLoad ModelGroup(ModelGroup modelGroup)
         /// Use to load ModelGroup by id
         /// </summary>
         /// <returns>ModelGroupLoad</returns>
-        /// <param name="modelId">Model identifier.</param>
-        public ModelGroupLoad ModelId(int modelId)
+        /// <param name="modelId">The Model ID to load. Often the Selected Model ID for the Model Group</param>
+        public ModelGroupLoad SetId(int modelId)
         {
```

Comment:
If we are allowing default getters and setters. there is no need for this again.